### ISSUE TITLE
interpolate at other positions, like media query args, selectors, attr()

### ISSUE THREAD

**Issue by tkh44** [tkh44/emotion#36]

**Comment by jgoux** [comment]
Hello,

does this issue includes referencing other styled components ?

Like this : 
```js
const Page = styled.div`
  display: flex;
  flex-direction: column;
  & > ${AppBar} { flex: none; }
  & > ${Main} { flex: 1; }
`
```

**Comment by tkh44** [comment]
@jgoux this has been released in 4.1.6 🎉

You can do exactly that.

**Comment by mitchellhamilton** [comment]
Closed by #56 and #76
